packages/common/src/search/getQueryContentConfig.ts

```diff
@@ -0,0 +1,349 @@
+import { IArcGISContext } from "../ArcGISContext";
```

Comment:
@mjuniper This is the meat of the "config" stuff. The catalog version of this fn, delegates here